{
    "title": "A denial of service issue was addressed with improved validation. This issue is fixed in iOS 12.4, watchOS 5.3. A remote attacker may cause an unexpected application termination.",
    "score": 7.5,
    "severity": "HIGH",
    "cwe_id": "CWE-20",
    "cvss_vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
    "cpe_match": [
        "cpe:2.3:o:apple:watchos:*:*:*:*:*:*:*:*",
        "cpe:2.3:o:apple:iphone_os:*:*:*:*:*:*:*:*"
    ],
    "cve_source": [
        "NVD"
    ],
    "ttpMapping": [
        {
            "techniqueId": "T1033",
            "confidence": "5.3",
            "techniqueName": "System Owner/User Discovery"
        },
        {
            "techniqueId": "T1083",
            "confidence": "2.1",
            "techniqueName": "File and Directory Discovery"
        },
        {
            "techniqueId": "T1123",
            "confidence": "3.5",
            "techniqueName": "Audio Capture"
        },
        {
            "techniqueId": "T1222",
            "confidence": "2.7",
            "techniqueName": "File and Directory Permissions Modification"
        },
        {
            "techniqueId": "T1489",
            "confidence": "2.0",
            "techniqueName": "Service Stop"
        },
        {
            "techniqueId": "T1505.003",
            "confidence": "5.2",
            "techniqueName": "Web Shell"
        },
        {
            "techniqueId": "T1543",
            "confidence": "4.4",
            "techniqueName": "Create or Modify System Process"
        }
    ]
}
